code/game/objects/items/weapons/cigs.dm

```diff
@@ -219,10 +219,6 @@ LIGHTERS ARE IN LIGHTERS.DM
 /obj/item/clothing/mask/cigarette/handroll
 	name = "hand-rolled cigarette"
 	desc = "A roll of tobacco and nicotine, freshly rolled by hand."
-	icon_state = "hr_cigoff"
-	item_state = "hr_cigoff"
```

Comment:
This is the only part that is not working. Both the icon_state and icon_on sprites are present and working, only the item_state is missing.